Library/Homebrew/cask/lib/hbc/cli/search.rb

```diff
@@ -21,7 +21,7 @@ def self.search(*arguments)
       partial_matches = Hbc::CLI.nice_listing(Hbc.all_tokens).grep(%r{#{search_regexp}}i)
     else
       # suppressing search of the font Tap is a quick hack until behavior can be made configurable
-      all_tokens = Hbc::CLI.nice_listing Hbc.all_tokens.reject { |t| %r{^caskroom/homebrew-fonts/}.match(t) }
+      all_tokens = Hbc::CLI.nice_listing Hbc.all_tokens.reject { |t| %r{^caskroom/fonts/}.match(t) }
```

Comment:
Looks like the filter was added in https://github.com/caskroom/homebrew-cask/pull/3025. Not sure why fonts would be too noisy unless you just search `font`. Probably fine to remove this.